code/src/layouts/intro-banner/IntroBanner.tsx

```diff
@@ -18,10 +18,7 @@ const IntroBanner = () => {
 
   return (
     <AppBox
-      className="spa-banner-intro"
-      style={{
-        backgroundImage: `url(${introBannerImages[currentImageIndex]})`
-      }}
+      className={`spa-banner-intro spa-banner-intro--image-${currentImageIndex}`}
```

Comment:
I think `spa-banner-intro__image` would be correct according to BEM methodology. `--` just for modifiers, not elements

Reply:
> I think `spa-banner-intro__image` would be correct according to BEM methodology. `--` just for modifiers, not elements

Sure. In my case it's a modifier that indicates a variation of the `spa-banner-intro` block based on the current image index

`__` used for sub elements I guess. It's a main wrapper as well why should to use subelements ?